src/evict/evict_lru.c

```diff
@@ -248,8 +248,16 @@ __evict_queue_full(WT_EVICT_QUEUE *queue)
 }
 
 /*
+ * NO_AUTO_FORMAT
+ *
  * __wt_evict_server_wake --
- *     Wake the eviction server thread.
+ *     This function wakes the eviction server thread. The eviction server typically sleeps for some
+ *     time when cache usage is below the target thresholds. When the cache is expected to exceed
+ *     these thresholds, this function is used to nudge the eviction server to wake up and resume
```

Comment:
```suggestion
 *     these thresholds, callers can nudge the eviction server to wake up and resume
```

Reply:
Updated in [73744c3](https://github.com/wiredtiger/wiredtiger/pull/11096/commits/73744c3a2b3e6a9811054165f507c2971025785e)